transactions/btc.ts

```diff
@@ -675,8 +677,8 @@ export async function signOrdinalSendTransaction(
 
 
   const filteredUnspentOutputs = [
-    ...filterUtxos(unspentOutputs, addressOrdinalsUtxos!),
-    ...filterUtxos(addressOrdinalsUtxos!, unspentOutputs),
+    ...filterUtxos(unspentOutputs, addressOrdinalsUtxos),
+    ...filterUtxos(addressOrdinalsUtxos, unspentOutputs),
```

Comment:
Doesn't this just give you `unspentOutputs` + `addressOrdinalsUtxos`? Which means the ordinals would still be in the UTXO set

Reply:
it does the opposite actually so it filters the first arr wit the content of the second one and then filters the second arr with the content of the first one the only way this could break is if the user have more ordinals utxos than utxos all-together which is not possible but basically the second filter is redundant